pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -93,28 +97,56 @@ func (admitter *KubeVirtUpdateAdmitter) validateWorkloadPlacementUpdate() ([]met
 	return []metav1.StatusCause{}, nil
 }
 
-func getAdmissionReviewKubeVirt(ar *v1beta1.AdmissionReview) (new *v1.KubeVirt, old *v1.KubeVirt, err error) {
+func getAdmissionReviewKubeVirt(ar *v1beta1.AdmissionReview) (new *v1.KubeVirt, old *v1.KubeVirt, isUpdate bool, err error) {
```

Comment:
This function uses a mixture of both names returned values and in the body explicitly returns others. For example, `isUpdate` is not used.

Please choose one or the other, but not a mix as it harder to read what is really going on.

Another problem is that it returns way too many arguments, which is usually a sign that it does too many things.
Per what I see, if `old` is `nil`, it means that nothing got updated and the new bool is redundant. Can it be used at the caller instead?